design-documents/graph-ql/coverage/store-pickup.graphqls

```diff
@@ -1,12 +1,15 @@
 type Query {
     pickupLocations (
-        filter: PickupLocationFilterInput,
+        area: AreaInput,
+        filters: PickupLocationFilterInput,
+        sort: PickupLocationSortInput,
         pageSize: Int = 20,
-        currentPage: Int = 1
+        currentPage: Int = 1,
+        productsInfo: [ProductInfoInput]
```

Comment:
Does this field exist?

Reply:
It is going to be delivered in https://github.com/magento/inventory/pull/3161